loadtest/tests.go

```diff
@@ -422,6 +422,10 @@ var standardUserEntity UserEntity = UserEntity{
 			Item:   actionDisconnectWebsocket,
 			Weight: 4,
 		},
+		{
+			Item:   actionMoreChannels,
+			Weight: 4,
```

Comment:
I've picked this somewhat arbitrarily. I'd like to revisit all of these constants by extracting metrics from pre-release or elsewhere.